roles/microshift_generate_iso/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+- import_tasks: "{{ microshift_generate_iso_action }}.yml" # noqa name[missing]
```

Comment:
Could you add some validation for the variable, to only allow expected values.

Also, is there any reason why to import and not include the task? Generally is preferred to include unless there's a particular reason on the import.

Reply:
No particular reason, I will do the change